rasa/telemetry.py

```diff
@@ -759,6 +760,12 @@ def track_data_split(fraction: float, data_type: Text) -> None:
     _track(TELEMETRY_DATA_SPLIT_EVENT, {"fraction": fraction, "type": data_type})
 
 
+@ensure_telemetry_enabled
+def track_data_augment() -> None:
```

Comment:
can you please update this file https://github.com/RasaHQ/rasa/blob/a6fd7c0deb243b3c6b24af3e265e16dec9f054dc/docs/docs/telemetry/events.json#L4 with the new telemetry event

Reply:
Is there any internal documentation available specifying what should go into the telemetry event?

Reply:
For PRs of this size there would usually be an implementation proposal in which you'd discuss what you want to track. Did you double check with Ty and Fede?

Reply:
I don't recall that we've discussed any aspects around telemetry. I guess we're primarily interested whether the feature is used _per se_. @dakshvar22 do you have any preferences on what we want to track for it?

Reply:
Since this is an experimental feature, at the simplest level we want to track the usage of the CLI command, as in how many users are using the command and at what frequency. Anything above that would be an overkill at this point.

Reply:
I have added an event without properties. LMK if this is suitable for just tracking whether or not the feature is used.